tests/testthat/test-utf8.R

```diff
@@ -0,0 +1,16 @@
+
+test_that("UTF-8 file names", {
+  skip_on_cran()
```

Comment:
Why did you need to skip this on CRAN?

Reply:
No specific reason, but they are updating a lot of things wrt encodings nowadays, and a couple of weird things have popped up recently.

Reply:
OTOH it would make sense to test brio on R-devel Windows, on the CI.

Reply:
I would prefer not to skip it unless we have a specific reason.

Sure we can test on R-devel for windows as well.

Reply:
Yeah, it is up to you, really. If we don't skip, then make sure you test on Solaris and a non-UTF-8 Linux as well before submitting to CRAN, just in case. These platforms might not work with non-ASCII characters at all.

Or we can make it Windows-only.